### PR TITLE
Player notification improvements

### PR DESCRIPTION
**Pull request by smashblu**
This PR will properly notify the player for events and what bets are placed. Primarily, there will be two main changes:

1. Betting chips will be displayed on the game board
2. Pop ups with descriptive messages will appear in the lower left

### CODE REVIEW

**Comment by smashblu** [comment]
I wonder if it would be better to incorporate secondaryBetsObj{} within chipSet() to reduce redundancy

File: resources/scripts.js [hunk 1]
```diff
@@ -247,28 +195,42 @@ function payOut(isprimary, win, amount) {
         if (win === true) {
```
Comment (smashblu) [inline resources/scripts.js:195]: payOut function needs to be refactored to not include displayMessage function

Comment (smashblu) [reply resources/scripts.js]: Changed function so that only primary wins display messages

File: index.html [hunk 2]
```diff
@@ -49,11 +49,7 @@ <h1 class="modal-title fs-5">Welcome to the Craps Table!</h1>
             </div>
             <h2></h2><img src="images/logo.png"> Browser Craps</h2>
         </div>
-        <div class="gameboard">
```
Comment (VeysonD) [inline index.html:52]: Good catch

**Review by VeysonD — APPROVED** [review]
LGTM

**Pull request merged**